ambry-clustermap/src/test/java/com/github/ambry/clustermap/MockHelixParticipant.java

```diff
@@ -100,11 +101,18 @@ public void participate(List<AmbryStatsReport> ambryStatsReports, AccountStatsSt
   }
 
   @Override
-  public boolean setReplicaSealedState(ReplicaId replicaId, boolean isSealed) {
-    if (isSealed) {
-      sealedReplicas.add(replicaId);
-    } else {
-      sealedReplicas.remove(replicaId);
+  public boolean setReplicaSealedState(ReplicaId replicaId, ReplicaSealStatus replicaSealStatus) {
+    switch (replicaSealStatus) {
+      case SEALED:
+        sealedReplicas.add(replicaId);
+        partiallySealedReplicas.remove(replicaId);
+        break;
+      case PARTIALLY_SEALED:
+        partiallySealedReplicas.add(replicaId);
```

Comment:
Are we supposed to add sealedReplicas.remove(replicaId)?

Reply:
fixed.